perses/app/fah_generator.py

```diff
@@ -40,18 +40,20 @@
                              'lambda_torsions': x}
 
 
-def make_neq_integrator(nsteps_eq, nsteps_neq, neq_splitting, alchemical_functions = DEFAULT_ALCHEMICAL_FUNCTIONS, **kwargs):
+def make_neq_integrator(nsteps_eq=1000, nsteps_neq=1000, neq_splitting='V R H O R V', timestep=4.0 * unit.femtosecond, alchemical_functions = DEFAULT_ALCHEMICAL_FUNCTIONS, **kwargs):
```

Comment:
If you're using 4 fs timesteps and we want 1 ns equilibrium and nonequilibrium dwell periods, both `nsteps_eq` and `nsteps_neq` should be `250000`.